### DISCUSSION TITLE
Cannot switch to remote Borg backup

### DISCUSSION THREAD

**Discussion by jalphad** [nextcloud/all-in-one#5653] [Questions] [answered]
### Steps to reproduce
1. Update an existing Nextcloud deployment (with local backups configured) to v10.0.0 through the beta channel
2. There is no option in the AIO interface to configure backups to a remote Borg instance

### Expected behavior 
I expect to be able to also switch my existing deployment to use a remote Borg repository for backup

### Actual behavior 
No options to reconfigure backups.
Note that it does work normally if I start from scratch.

### Other information
Can be provided on request

**Comment by szaimen** [comment]
Cc @timdiels maybe we should document this somewhere? https://github.com/nextcloud/all-in-one/discussions/5593

**Reply by hannahdiels** [reply]
@szaimen Yes, I'll add something to the readme.

**Reply by hannahdiels** [reply]
I'm actually unsure how people have been able to switch because [as I originally expected](https://github.com/nextcloud/all-in-one/discussions/5593#discussioncomment-11284844) the Reset backup location section is hidden after the first backup has been made.

**Reply by hannahdiels** [reply]
Originally discussed how to migrate here https://github.com/nextcloud/all-in-one/discussions/5593#discussioncomment-11285112 (I tried `docker compose up -d` but still no reset section)

I'm tempted to remove the if around "Reset backup location" so it remains available.

**Comment by jalphad** [comment, marked as answer]
Thanks @szaimen for pointing me to that ticket. I somehow missed it earlier. Using that I was able to reset my backup location to a remote one. [This](https://github.com/borgbackup/borg/issues/3867) Github issue (and the related issues) for Borg suggests that simply copying the data is advised against so I just started a new backup in the new location.
